sdk/data/azcosmos/cosmos_example_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"os"
+)
+
+// This example shows you how to get started using the Azure Cosmos DB SDK for Go.
+// NewCosmosClient creates a new instance of CosmosClient with the specified values. It uses the default pipeline configuration.
+func Example() {
+
+	endpoint, _ := os.LookupEnv("SOME_ENDPOINT")
+	key, _ := os.LookupEnv("SOME_KEY")
+
+	// Create new Cosmos DB client.
+	cred, _ := NewSharedKeyCredential(key)
+	client, err := NewCosmosClient(endpoint, cred, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// All operations for Go operate on a context.Context, allowing you to control cancellation/timeout.
+	ctx := context.Background()
+
+	// ===== 1. Creating a database =====
+
+	databaseName := CosmosDatabaseProperties{Id: "databaseName"}
+	database, err := client.CreateDatabase(ctx, databaseName, nil, nil)
+	log.Panic(err)
+
+	// This example showcases several common operations to help you get started, such as:
+	// ===== 2. Creating a container =====
+
+	properties := CosmosContainerProperties{
+		Id: "aContainer",
+		PartitionKeyDefinition: PartitionKeyDefinition{
+			Paths: []string{"/id"},
+		},
+		IndexingPolicy: &IndexingPolicy{
```

Comment:
Let's start with a simple case of not defining the IndexingPolicy (should use the service default)